third_party/move/move-compiler-v2/tests/checking/typing/dummy_field.exp

```diff
@@ -36,3 +36,29 @@ module 0x42::test {
         }
     }
 } // end 0x42::test
+
+// -- Sourcified model before bytecode pipeline
+module 0x42::test {
+    struct T has drop, store, key {
```

Comment:
This perhaps a good approach, but it would require a program analysis since via a Pattern the field maybe be bound; it is just that the local is never used. This happens when coming from bytecode where there is no `_` wildcard. Correct would it to just always generate the field, it just looks ugly.  Lets come back, I added a TODO